packages/lab/src/ProgressBar/ProgressBar.js

```diff
@@ -9,6 +9,8 @@ import { withStyles } from "@material-ui/core";
 
 import styles from "./styles";
 
+const statusCategories = { inProgress: "inProgress", completed: "completed", error: "error" };
```

Comment:
You can include an `object.freeze({ inProgress: "inProgress", completed: "completed", error: "error" };)` to ensure this object remains consistent